cli/flox-rust-sdk/src/providers/services.rs

```diff
@@ -274,31 +269,53 @@ impl ProcessStates {
             .map(|state| state.name.clone())
             .collect()
     }
+}
 
-    fn fmt_table(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
-        writeln!(f, "{:<20} {:<10} {:>8}", "NAME", "STATUS", "PID")?;
-        for proc in &self.0 {
-            writeln!(f, "{:<20} {:<10} {:>8}", proc.name, proc.status, proc.pid)?;
-        }
-        Ok(())
+#[derive(Clone, Debug, Serialize)]
+/// Simplified version of ProcessState for display in the CLI.
+pub struct ProcessStateDisplay {
+    name: String,
+    status: String,
+    pid: u64,
+}
+
+#[derive(Clone, Debug, Serialize)]
+/// Simplified version of ProcessStates for display in the CLI.
+pub struct ProcessStatesDisplay(Vec<ProcessStateDisplay>);
+
+impl From<ProcessStates> for ProcessStatesDisplay {
+    fn from(procs: ProcessStates) -> Self {
+        ProcessStatesDisplay(
+            procs
+                .0
+                .into_iter()
+                .sorted_by_key(|proc| proc.name.clone())
```

Comment:
qeustion: this is one of the fun ones that doesnt support returning refs for comparison right?